sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1731,6 +1731,17 @@ public boolean isProfilingEnabled() {
         || getProfilesSampler() != null;
   }
 
+  /**
+   * Returns if continuous profiling is enabled. This means that no profile sample rate has been
+   * set.
+   *
+   * @return if continuous profiling is enabled.
+   */
+  @ApiStatus.Internal
+  public boolean isContinuousProfilingEnabled() {
```

Comment:
hm, so this means setting `profilesSampleRate` to 0 would disable both the continuous and the tx-based profilers? I guess we should document this very well as this may be confusing

Reply:
[yep](https://www.notion.so/sentry/Continuous-Profiling-SDK-46f8f4ce691a4fedbcc39c7f1e1e740e?pvs=4#12f8b10e4b5d803e959bca2b9d533afc)
It's already pointed out in the [docs](https://docs.sentry.io/platforms/apple/profiling/#continuous-profiling) (Android docs will be mostly the same): `If you had previously set SentryOptions.profilesSampleRate or SentryOptions.profilesSampler to use transaction-based profiling, then remove those lines of code from your configuration.`
I also added a log message when the startProfile() is called and a `profilesSampleRate` is set, to avoid this confusion in [p5](https://github.com/getsentry/sentry-java/pull/3844)